pom.xml

```
@@ -87,8 +87,8 @@
     <!-- By default we do not create *-tests.jar. Set to false to allow other plugins to depend on test utilities in yours, using <classifier>tests</classifier>. -->
     <no-test-jar>true</no-test-jar>
 
-    <node.version>4.0.0</node.version>
-    <npm.version>2.13.1</npm.version>
+    <node.version>16.17.1</node.version>
+    <npm.version>8.19.2</npm.version>
```

Is anyone actually relying on the version from the parent POM? I glanced at a few example consumers, and all of them were defining their own versions. If we want people to use the versions from the parent, then I think it makes sense to keep it up-to-date. But if we want people to define their versions, then I think it would be better to force them to explicitly (rather than implicitly via the ancient versions currently defined) via something like this:

```suggestion
    <node.version>you-must-override-the-node.version-property</node.version>
    <npm.version>you-must-override-the-npm.version-property</npm.version>
```

Since I think Node and NPM versions usually need package or dependency updates that are tied to the specific repository and need to be tested individually, I think it likely makes more sense to get `plugin-pom` out of the business of declaring specific versions and forcing consumers to take the matter completely into their own hands, as seems to be the case in practice already.

---

One of the reasons why (almost) nobody uses the preset values is, that those are dated and wouldn't function with up-to-date NPM dependencies anyway, hence we even need to overwrite them in core itself.

I'd assume, maintaining these couple of lines properly to provide up-to-date versions, would bring back people to the plugin-pom version, rather than defining custom versions.

---

Do not make assumptions. If you desire for that to be the case, enumerate the consumers and file draft PRs updating them to these versions, and test the results. If you do not wish to do a large project like that, then go the easier route and remove the versions from `plugin-pom`.